php/ubiquity/composer.json

```diff
@@ -1,12 +1,12 @@
 {
 	"require" : {
 		"php" : "^7.1",
-		"phpmv/ubiquity" : "~2.3.2"
+		"phpmv/ubiquity" : "~2.3.2",
+		"phpmv/ubiquity-devtools": "~1.2.10"
```

Comment:
As I have already explained to you (see https://github.com/the-benchmarker/web-frameworks/pull/1944#discussion_r341587296), I disapprove this practice, since the **devtools** can be installed globally, without overloading the **composer** autoloading for the project.